src/request/import/ImportWords.js

```diff
@@ -241,6 +272,8 @@ class ImportWords {
                     wordsExported: true,
                 };
                 this._keyResults.push(result);
+
+                sessionStorage.setItem(ImportApi.SESSION_STORAGE_KEY_PREFIX + result.keyId, '');
```

Comment:
I think using the empty string here is wrong, as it will be evaluated as falsy in `releaseKey` in `IframeApi`.
When changing it, I think force pushing would be good.